packages/providers/src/util.ts

```diff
@@ -95,3 +95,31 @@ export const calculateTransactionFee = ({
     fee,
   };
 };
+
+const DEFAULT_BLOCK_EXPLORER_URL = 'https://fuellabs.github.io/block-explorer-v2';
+
+/**
+ * Builds a block explorer url based on and the given path, block explorer URL and provider URL
+ */
+export const buildBlockExplorerUrl = ({
+  blockExplorerUrl,
+  path,
+  providerUrl,
+}: {
+  blockExplorerUrl?: string;
+  path: string;
+  providerUrl?: string;
```

Comment:
we should support receiving both `blockExplorerUrl` and `providerUrl` without the protocol.
if we detect that the user didn't provide protocol (`://`) in the string, we auto prefix the input with `https://`

to increase DX / flexibility, if the user informs `blockExplorerUrl: 'explorer.fuel.sh'`, it should just work

Reply:
@LuizAsFight gotcha, adding this now